.github/CODEOWNERS

```diff
@@ -26,3 +26,4 @@
 /tools/mock-service-host/        @changlong-liu @tadelesh
 /tools/sdk-testgen/              @changlong-liu @tadelesh
 /tools/js-sdk-release-tools/     @weidong @qiaozha @MaryGao
+/tools/keyvault-mock-attestation @maorleger
```

Comment:
Thanks for adding the codeowners entry. Sorry I just caused a conflict when I fixed the error in the existing file.

Reply:
No worries, it's easy to fix